hal/halx86/generic/halinit.c

```diff
@@ -14,31 +14,10 @@
 
 /* GLOBALS *******************************************************************/
 
-BOOLEAN HalpPciLockSettings;
+UCHAR HalpInitLevel = 0xFF;
 
 /* PRIVATE FUNCTIONS *********************************************************/
 
-CODE_SEG("INIT")
-VOID
-NTAPI
-HalpGetParameters(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
-{
-    PCHAR CommandLine;
-
-    /* Make sure we have a loader block and command line */
-    if ((LoaderBlock) && (LoaderBlock->LoadOptions))
-    {
-        /* Read the command line */
-        CommandLine = LoaderBlock->LoadOptions;
-
-        /* Check if PCI is locked */
-        if (strstr(CommandLine, "PCILOCK")) HalpPciLockSettings = TRUE;
-
-        /* Check for initial breakpoint */
-        if (strstr(CommandLine, "BREAK")) DbgBreakPoint();
-    }
-}
```

Comment:
Why do we have this function duplicated 3 times in minihal, pic, and xbox HAL now instead of keeping it here?

Reply:
Because I added an variant for this function (for APIC) and a conflict appeared
![HalpGetParameters](https://user-images.githubusercontent.com/17290914/111367967-3608f700-868d-11eb-84ea-69b485de7f4f.png)